package.json

```diff
@@ -1,11 +1,10 @@
 {
   "name": "suomifi-ui-components",
-  "version": "0.4.1",
+  "version": "0.5.0-alpha.1",
```

Comment:
Is this a good idea to change the version with this PR? I'd propose to keep it as is and make a separate PR and a tag for 0.5.0.

Reply:
Good point. Didn't notice the version number increase that we made for the alpha release when making PR.

Let's do like you suggested; keeping it at the previous `0.4.1` in this PR and increase it in the PR of `release/v0.5.0` little later.